.github/workflows/python.yml

```diff
@@ -114,5 +114,5 @@ jobs:
           file: Dockerfile.s390x.test
           tags: ${{ steps.meta.outputs.tags }}
           labels: ${{ steps.meta.outputs.labels }}
-          cache-from: type=registry,ref=registry.internal.huggingface.tech/safetensors/s390x:cache,mode=max
-          cache-to: type=registry,ref=registry.internal.huggingface.tech/safetensors/s390x:cache,mode=max
+          cache-from: type=registry,ref=registry.internal.huggingface.tech/safetensors/s390x:cache,mode=min
+          cache-to: type=registry,ref=registry.internal.huggingface.tech/safetensors/s390x:cache,mode=min
```

Comment:
Curious to hear what the motivation behind this change was

Reply:
The tests were failing, deleting cache made them work again. I therefore modified this and deleted the cache again. Didn't look too hard into it.